llms/mlx_lm/models/olmo.py

```diff
@@ -0,0 +1,159 @@
+from dataclasses import dataclass
+from typing import Dict, Optional, Tuple, Union
+
+import mlx.core as mx
+import mlx.nn as nn
+
+from .base import BaseModelArgs
+
+try:
+    import hf_olmo
```

Comment:
Just curious, what is the reason we need to import hf_olmo? It doesn't seem to be used.

Reply:
Well yes, I added that because the tokenizer needs it (I am not actually sure if it needs it but it tries to import it and it won't load otherwise). And I didn't want to add this as a general dependency..which seemed a bit overkill.

Reply:
makes sense. Thanks for clarifying it.